src/frontend/shared/components/NavigationBar.tsx

```diff
@@ -1,12 +1,8 @@
 import * as React from 'react';
-import MenuItem from "./MenuItem";
-import {FriendBannerProps} from "./FriendBanner";
+import NavigationItems from "./NavigationItems";
+import {IUserProps} from "../interfaces/IUserProps";
 
-interface SidebarProps {
-    user_info: FriendBannerProps;
-}
-
-class Sidebar extends React.Component<SidebarProps, {}> {
+class NavigationBar extends React.Component<IUserProps, {}> {
```

Comment:
Reason for renaming: I think naming components based on their purpose rather than what the component is implemented as is more important. Since we have two side bars, it might be easy to get confused with the right side bar.